src/Expanders.jl

```diff
@@ -572,7 +572,11 @@ function Selectors.runner(::Type{ExampleBlocks}, x, page, doc)
 
     # Only add content when there's actually something to add.
     isempty(input)  || push!(content, Markdown.Code("julia", input))
-    isempty(output) || push!(content, output)
+    if result === nothing
+        push!(content, Markdown.Code(Documenter.DocTests.sanitise(buffer)))
```

Comment:
I guess we should only `push!` to `content` if `Documenter.DocTests.sanitise(buffer)` returns something non-empty?

Reply:
This is also consistent with how it was before.